### ISSUE TITLE
amplify init failed to complete and created an undeletable app

### ISSUE THREAD

**Issue by joscarras** [aws-amplify/amplify-cli#6156]
**Describe the bug**
I have been trying to learn amplify, and today in one of my tests `amplify init` failed to complete due to:
**DeploymentBucket amplify-test-dev-155008-deployment already exists**

After that I was unable to do anything so I deleted the folder and went into the amplify console to delete that test app (so I can restart) however it failed with the following error:
**Removing backend envs for app d3c4lmqph8kggw failed**

I tried to delete the dev env directly however that resulted in the following error:
**Deleting backend env: dev from app: d3c4lmqph8kggw failed**

Is there anything I can do to remove this app right now?

**To Reproduce**
1. Go to 'AWS Amplify Console'
2. Select the app you wish to delete
2. Click on 'Actions' and select 'Delete'
3. Type 'delete' on the input and click on 'Delete'
4. See error "Removing backend envs for app d3c4lmqph8kggw failed"

**Expected behavior**
The test app to be deleted

**Screenshots**
![Screenshot_2020-12-04 AWS Amplify Console](https://user-images.githubusercontent.com/5412596/101183008-8f599700-3657-11eb-8313-c1cb2588a214.png)

**Desktop:**
 - OS: manjaro
 - Browser firefox
 - Version 83.0

**Comment by yuth** [comment]
@joscarras could you try to delete the application from Amplify console again and see if it gets deleted now. We had some issue in the service earlier which is fixed now.

Could you also share the steps that you followed when you got the error ` DeploymentBucket amplify-test-dev-155008-deployment already exists`. The bucket names are generated to such that they are unique

**Comment by yuth** [comment]
> The same situation.
> I tried to create first demo project with Amplify CLI, but in init process my Mac was crashed! I tries it several times and always was this result. And now I have several non-deletable demo apps :((

@makiyov  Did your Mac crash or the CLI crash? And did you try to init the app in the same folder after the first crash

**Comment by makiyov** [comment]
> > The same situation.
> > I tried to create first demo project with Amplify CLI, but in init process my Mac was crashed! I tries it several times and always was this result. And now I have several non-deletable demo apps :((
> 
> @makiyov Did your Mac crash or the CLI crash? And did you try to init the app in the same folder after the first crash

My Mac crash! I have iMac with macOS Big Sur 11.0.1. 
But, my MacBook Pro with macOS Catalina 10.15.7 works fine!

And yes, I try to init this app several time, result was the same - crash.

A little bit later I will update iMar to Big Sur 11.1 11.1 and check again.

**Comment by yuth** [comment]
Do you have any stack trace from the crash? What version of NodeJs is being used? and what is the version of Amplify CLI do you have installed?

**Comment by makiyov** [comment]
> Do you have any stack trace from the crash? What version of NodeJs is being used? and what is the version of Amplify CLI do you have installed?

So, now I have already updated all of these components to: macOS Big Sur 11.1, Node v15.4.0, Amplify CLI 4.40.0.
And you know, after that everything works great!!!
Thanks a lot for your attention!
Having latest versions is so important :))

**Comment by kaustavghosh06** [comment]
@joscarras The delete issue from the Amplify Console should be working now.
@makiyov Glad upgrading the CLI version worked for you :)

**Comment by joscarras** [comment]
Apologies for the late response!

@yuth I was messing really hard around for testing / learning so I don't remember the steps I did to get the error, however it was probably something I did in my end and not something with amplify. If I remember correct I was messing with the build folder.

@kaustavghosh06 I was able to delete the project just fine today, so yes I can confirm this issue is resolved!

Thank you both for your assistance!
